util/import/BUILD.bazel

```diff
@@ -70,20 +71,20 @@ config_setting(
     },
 )
 
-sh_binary(
+cc_binary(
```

Comment:
This being a `sh_binary` added an unnecessary reliance on bash for windows as well as required executable files be present in release artifacts, which there are none of.